src/lms-ds-loader/edfi_lms_ds_loader/db_operations/MSSQL_sql_builder.py

```diff
@@ -187,38 +187,45 @@ def insert_new_records_to_production_for_assignment_and_user_relation(
 
 
 def insert_new_records_to_production_for_attendance_events(
-        table: str,
         insert_columns: str,
         select_columns: str) -> str:
     return f"""
 INSERT INTO
-    lms.{table}
+    lms.LMSUserAttendanceEvent
 (
-    AssignmentIdentifier,
-    LMSUserIdentifier,{insert_columns}
+    LMSSectionIdentifier,
+    LMSUserIdentifier,
+    LMSUserLMSSectionAssociationIdentifier,{insert_columns}
```

Comment:
We overlooked a column? And now you're fixing the bug?

Reply:
I was returning the wrong sql for this method and I fixed it

Reply:
This also answers the previous comment on why I hardcoded the LMSUserAttendanceEvent, it does not have to be dynamic for this method and I am just fixing it, I realized of the error while fixing the tests